web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+odoo.define('web_ir_actions_act_view_reload.ir_actions_act_view_reload', function (require) {
+"use strict";
+
+    var ActionManager = require('web.ActionManager');
+
+    ActionManager.include({
+        ir_actions_act_view_reload: function (action, options) {
+            if (this.inner_widget && this.inner_widget.active_view && this.inner_widget.active_view.controller)
+                this.inner_widget.active_view.controller.reload();
+            return $.when();
+        },
```

Comment:
trailing comma might break on some stupid browser :)

Reply:
Fixed :)

Reply:
@simahawk small off topic: when i need to add oca-dependency for a module, how do I do that in case my dependency module is waiting in another pull request?

Reply:
you can specify to use your custom fork and branch

https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt

:wink: